getting-started/events.md

```diff
@@ -0,0 +1,268 @@
+
+```js
+{
+  "id": "0affb00a-82d6-4e07-ae61-56ba5c36f743", // generated by server side
+  "custom_id": "12312.301231238591134",
+  "order": 4, // generated by server side
+  "type": "filled_form",
+  "author_id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
+  "timestamp": 1473433500,  // generated by server side
+  "properties": {
+    // "Properties" object
+  }
+  "fields": [
+    // "Field" objects
+  ]
+}
+```
+
+There are 5 different types of field from filled form, all of these types can be mixed together in one filled form event.
+
+`text`:
+
+```js
+  {
+    "type": "text",
+    "name": "name",
+    "label": "Your name",
+    "required": true,
+    "value": "Jan Kowalski"
+  }
+```
+
+`email`:
+
+```js
+  {
+    "type": "email",
+    "name": "email",
+    "label": "Your email",
+    "required": true,
+    "value": "jan.kowalski@gmail.com"
+  }
+```
+
+`radio`:
+
+```js
+  {
+    "type": "radio",
+    "name": "purpose",
+    "label": "Chat purpose",
+    "required": true,
+    "options": [{
+      "label": "Support",
+      "value": "support",
+      "checked": true
+    },
+    {
+      "label": "Sale",
+      "value": "sale",
+      "checked": false
+    }],
+  }
+```
+
+`checkbox`:
+
+```js
+  {
+    "type": "checkbox",
+    "name": "industry",
+    "label": "Company industry",
+    "required": true,
+    "options": [{
+      "label": "automotive",
+      "value": "automotive",
+      "checked": true
+    }, {
+      "label": "IT",
+      "value": "it",
+      "checked": true
+    }],
+  }
+```
+
+`select`:
+
+```js
+{
+  "type": "select",
+  "name": "country",
+  "label": "Country",
+  "required": true,
+  "options": [{
+    "label": "USA",
+    "value": "usa",
+    "checked": false
+  }, {
+    "label": "Poland",
+    "value": "pl",
+    "checked": true
+  }, {
+    "label": "Poland",
+    "value": "pl",
+    "checked": false
+  }]
+}
+```
+
+
+### System message
+
+System message is event generated by server. It does not have "author_id", "custom_id" and "properties" fields.
+
+```js
+{
+  "id": "0affb00a-82d6-4e07-ae61-56ba5c36f743",
+  "order": 1,
+  "type": "system_message",
+  "timestamp": 1473433500,
+  "text": "Mike joined the chat",
+  "system_message_type": "agent_joined"
+}
+```
+
+System messages can be triggered by an user action within chat or by router (more about router messages in future Router section). Each action triggers system message with different `system_message_type` and has different `text` message:
+
+| system_message_type | text                             |
+|---------------------|----------------------------------|
+| `agent_joined`      | `<agent_name> joined the chat`   |
+| `agent_left`        | `<agent_name> left the chat`     |
+| `manual_archived`   | `<agent_name> archived the chat` |
+
+
+### File
+
+File event indicates that a file was uploaded. It can be only generated by server when [send file](../../customer-api/client-server#send-file) customer API method is called.
```

Comment:
File event indicates that a file was uploaded. It can be only generated by server when [send file](../../customer-api/client-server#send-file) customer API method is called. -> File event indicates that a file has been uploaded. It can be only generated by the server when the [send file](../../customer-api/client-server#send-file) customer API method is called.